colors.css

```diff
@@ -0,0 +1,4 @@
+$cream-can: #F5C85F
+$galery: #EEE
```

Comment:
you didn't delete this file :P

Reply:
it seems i did it in another branch, LOL.